src/Umbraco.Web.UI.Client/src/views/propertyeditors/datepicker/datepicker.html

```diff
@@ -38,7 +38,7 @@
             <p class="help-inline" ng-message="pickerError"><localize key="validation_invalidDate">Invalid date</localize></p>
         </div>
 
-        <p ng-if="model.config.offsetTime === '1' && serverTimeNeedsOffsetting && model.value" class="muted">
+        <p xng-if="model.config.offsetTime === '1' && serverTimeNeedsOffsetting && model.value" class="muted">
```

Comment:
What is `xng`? As far I know it isn't an attribute in older version of Angular.. maybe in newer versions of Angular?

Reply:
@bjarnef it was a test, to see if you need a cup of coffé. Well jokes aside it was a mistake that came along with the commit. It was a test to permeant see the offset value, do to testing where the problem was. I have removed it! Sorry for that